docs/book/src/getting-started/testdata/project/internal/controller/memcached_controller.go

```diff
@@ -89,7 +91,7 @@ func (r *MemcachedReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	}
 
 	// Let's just set the status as Unknown when no status is available
-	if memcached.Status.Conditions == nil || len(memcached.Status.Conditions) == 0 {
+	if memcached.Status.Conditions == nil || len(memcached.Status.Conditions) == 0 { // nolint:gosimple
```

Comment:
We should not ignore all lint checks
We should fix the scenarios in all possible cases where it is possible